Tests/MoyaTests/SignalProducer+MoyaSpec.swift

```diff
@@ -3,6 +3,7 @@ import Moya
 import ReactiveSwift
 import Nimble
 import Foundation
+import CoreGraphics
```

Comment:
It seems this PR is no longer required.
I just run the tests using `master` branch, latest version of Xcode, and all good.

<img width="702" alt="Screenshot 2022-12-14 at 10 10 27 PM" src="https://user-images.githubusercontent.com/12019206/207763993-c187244e-1447-42d0-9d5a-f749e0eefd7b.png">